src/launch_common.py

```diff
@@ -147,6 +147,12 @@ def args_validation(args, argparser):
         else:
             args.dry_run = args.dry_run_no_consumers
     else:
+        try:
```

Comment:
How those options impact each other and how they can be combined confuses me slightly